Core/Src/nero.c

```diff
@@ -26,22 +27,6 @@ static void send_mode_status()
 
 void set_mph(int8_t new_mph)
 {
-	//printf("mph %d", new_mph);
 	mph = new_mph;
-}
-
-osThreadId_t nero_monitor_handle;
-const osThreadAttr_t nero_monitor_attributes = {
-	.name = "NeroMonitor",
-	.stack_size = 32 * 8,
-	.priority = (osPriority_t)osPriorityHigh2,
-};
-
-void vNeroMonitor(void *pv_params)
-{
-	for (;;) {
-		send_mode_status();
-
-		osDelay(NERO_DELAY_TIME);
-	}
+	send_nero_msg();
```

Comment:
Its wierd sending nero message here and in the watchdog default task, however we do not need to do it in both tasks.  This will be sent every 25 milliseconds, which is fast but its ok so we can remove it in default task?

Reply:
My setup was everytime nero data was updated, a flag was set so that the nero monitor would run and send that data. Now we send a message every time data is updated. I only put the send func in the default task because peyton wants it to be sent continuously so that if the message is lost there will be another one coming.

Reply:
But it will be sent everytime any part of the message updates, which is a little bit extra.  I think we should have only one spot where the message is sent, here is OK.

Reply:
Or in the default.

Reply:
think we should see what happens with it just here..if its getting ost or peyton has issues we can keep it in both